docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: JetBrains SFTP
+description: How to use JetBrains IDE to browse remote host with Teleport
+h1: SFTP with JetBrains IDE
+---
+
+This guide explains how to use Teleport and JetBrains IDE to access files with SFTP.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- JetBrains IDE like PyCharm, IntelliJ, Goland etc. [Products](https://www.jetbrains.com/products/#type=ide)
+- One or more Teleport Nodes with Server Access enabled. If you have not yet
+done this, read the [Server Access Getting Started Guide](../getting-started.mdx) to learn how.
+
+## Step 1/3. First-time setup
+
+Configure your local SSH client to access Teleport Nodes, assigning the `--proxy` flag to the address of your Teleport Proxy Service (e.g., `mytenant.teleport.sh` for Teleport Cloud users).
+
+```code
+# log in to your proxy:
+$ tsh login --proxy proxy.foo.example.com --user alice
+
+# generate the OpenSSH config for the proxy:
+$ tsh config --proxy proxy.foo.example.com
+```
+
+Append the resulting configuration snippet into your SSH config file located
+in the path below:
+
+<Tabs>
+  <TabItem label="Linux/MacOS">
+    `$HOME/.ssh/config`
+  </TabItem>
+  <TabItem label="Windows">
+    `%UserProfile%\.ssh\config`
+
+    <Admonition type="warning">
+      If using PowerShell on Windows to write your SSH config, note that normal
+      shell redirection may write the file with the incorrect encoding. To
+      ensure it's written properly, try the following:
+
+      ```code
+      $ tsh.exe config | out-file .ssh\config -encoding utf8 -append
+      ```
+    </Admonition>
+  </TabItem>
+</Tabs>
+
+You should be able to connect to the desired node using following command, replacing `user` with the username you would like to assume on the node.
```

Comment:
```suggestion
You should be able to connect to the desired Node using following command, replacing `user` with the username you would like to assume on the Node:
```
Using "Node" so Teleport SSH Service instances are named consistently in the docs.